Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -2032,7 +2032,14 @@ InstanceInfoRecord UpdateInstanceInfo(InstanceInfoData data)
 
             objTransform.rotation = Quaternion.LookRotation(forward, upwards);
 
-            Debug.Assert(objTransform.localToWorldMatrix == newMat, "Matrices don't match!");
+#if DEBUG
+            var localMatrix = objTransform.localToWorldMatrix;
+            for (int x = 0; x < 3; x++) {
+                for (int y = 0; y < 3; y++) {
+                    Debug.Assert( Math.Abs(localMatrix[x, y] - newMat[x, y]) < 0.01f, "Matrices don't match!");
```

Comment:
Maybe we should use [Mathf.Approximately](https://docs.unity3d.com/ScriptReference/Mathf.Approximately.html) ?

Reply:
The tolerance in that is too small. We get rounding errors that are larger than that.